python.md

```diff
@@ -287,6 +287,75 @@ mycln.update_one({"name" : "foo"}, {"$set":{"age": 23}})
 # to delete a single document
 doc1 = mycln.delete_one({"name" : "foo"})
 ```
+## Python with MySQL
```

Comment:
We want to cover basics only in Cheat Sheets, this is advanced information. You can always create these as posts in OneCompiler

Reply:
If you permit, I can create a separate file for this module. This info is pretty basic tho and comes handy

Reply:
No please